File: objectivity/game_state.m
#import "game_state.h"

@implementation GameState
@synthesize players;

- (id) init
{
  self = [super init];

  [self setPlayers: [[NSMutableArray alloc] init]];
  return self;
}


- (void) addPlayer: (Player *)aPlayer
{
  [self.players addObject:aPlayer];
}
@end

File: ceepuhpuh/game_state.cpp
#include "game_state.h"


GameState::GameState()
{
	this->players = new PlayerList;
}


Player* GameState::add_player (Player *player)
{
	this->players->push_back (player);
	return player;
}

PlayerList* GameState::get_players ()
{
	return this->players;
}

